Dayanand Viveki/HTML/Assignment6.2/personalfulldetails.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Personal Full Details</title>
+    </head>
+    <body>
+        <div class="banner">
+            <div class="headline font_size_35">
+                <p>MI-11 Inter HTML Assesment</p>
+            </div>
+            <hr />
+            <div class="headline font_size_25 italic_font">
+                <p>Personal Details</p>
+            </div>
+            <div class="form_item">
+                <form action="">
+                    <span>
```

Comment:
remove all span tags no need of including lable and input tag inside them

Reply:
ok mdm